mod/wikirate_source/spec/set/type_plus_right/source/file-import_spec.rb

```diff
@@ -2,191 +2,185 @@
-      amazon_2015_metric_value_card = Card["Access to Nutrition Index+Marketing Score+Amazon.com, Inc.+2015+value"]
-      apple_2015_metric_value_card = Card["Access to Nutrition Index+Marketing Score+Apple Inc.+2015+value"]
-      expect(amazon_2015_metric_value_card.content).to eq("369")
-      expect(apple_2015_metric_value_card.content).to eq("689")
+      expect(Card.exists?'Access to Nutrition Index+Marketing Score+Amazon.com, Inc.+2015+link').to be false
+      expect(Card.exists?'Access to Nutrition Index+Marketing Score+Apple Inc.+2015+link').to be false
+      amazon_2015_metric_value_card = Card['Access to Nutrition Index+Marketing Score+Amazon.com, Inc.+2015+value']
+      apple_2015_metric_value_card = Card['Access to Nutrition Index+Marketing Score+Apple Inc.+2015+value']
+      expect(amazon_2015_metric_value_card.content).to eq('369')
+      expect(apple_2015_metric_value_card.content).to eq('689')
     end
   end
-  describe "while rendering import view" do
+  describe 'while rendering import view' do
 
-    it "shows field correctly" do
+    it 'shows field correctly' do
 
-      source_file_card = @source.fetch :trait=>:file
+      source_file_card = @source.fetch trait: :file
       html = source_file_card.format.render_import
 
-      expect(html).to have_tag("div", :with=>{:card_name=>"#{@source.name}+Metric"}) do
-        with_tag "input", :with=>{:class=>"card-content form-control",:id=>"card_subcards_#{@source.name}_Metric_content"}
+      expect(html).to have_tag("div", with: {card_name: "#{@source.name}+Metric"}) do
+        with_tag "input", with: {class: "card-content form-control",id: "card_subcards_#{@source.name}_Metric_content"}
       end
-      expect(html).to have_tag("div", :with=>{:card_name=>"#{@source.name}+Year"}) do
-        with_tag "input", :with=>{:class=>"card-content form-control",:id=>"card_subcards_#{@source.name}_Year_content"}
+      expect(html).to have_tag("div", with: {card_name: "#{@source.name}+Year"}) do
+        with_tag "input", with: {class: "card-content form-control",id: "card_subcards_#{@source.name}_Year_content"}
       end
-      expect(html).to have_tag("input", :with=>{:id=>"is_metric_import_update",:value=>"true",:type=>"hidden"})
-      expect(html).to have_tag("table", :with=>{:class=>"import_table"}) do
-        with_tag "tr" do
-          with_tag "input", :with=>{:type=>"checkbox",:value=>"43",:id=>"metric_values_Cambridge_"}
-          with_tag "td",:text=>"Cambridge"
-          with_tag "td" do
-            with_tag "input", :with=>{ :type=>"text", :name=>"corrected_company_name[Cambridge]" }
+      expect(html).to have_tag("input", with: {id: "is_metric_import_update",value: "true",type: 'hidden'})
+      expect(html).to have_tag("table", with: {class: 'import_table'}) do
+        with_tag 'tr' do
+          with_tag "input", with: {type: "checkbox",value: "43",id: 'metric_values_Cambridge_'}
+          with_tag "td",text: 'Cambridge'
```

Comment:
Space missing after comma.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.